frontend/e-buddy/src/app/(demo)/bots/view/page.tsx

```
@@ -0,0 +1,233 @@
+            <div className="col-span-3 text-right">Actions</div>
+          </div>
+
+          {/* Rows */}
+          {sortedBots.length ? (
+            sortedBots.map((bot) => (
+              <div
+                key={bot.id}
+                className="grid grid-cols-1 sm:grid-cols-12 gap-4 items-center p-4 hover:bg-gray-50 transition-colors cursor-pointer"
+                onClick={() => router.push(`/bots/${bot.id}`)}
+              >
+                <div className="col-span-1 sm:col-span-4 flex items-center space-x-3">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)